packages/paste-core/components/editable-code-block/src/EditableCodeBlockHeader.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import {Box, safelySpreadBoxProps, type BoxProps} from '@twilio-paste/box';
+import type {asTags} from '@twilio-paste/heading';
+
+export interface EditableCodeBlockHeaderProps extends Partial<Omit<HTMLHeadingElement, 'children'>> {
+  children: string;
+  element?: BoxProps['element'];
+  as?: asTags;
+}
+
+export const EditableCodeBlockHeader = React.forwardRef<HTMLHeadingElement, EditableCodeBlockHeaderProps>(
```

Comment:
is it just me or are these not showing up in any of the examples

Reply:
+1

Reply:
If Shadi is using the storybook previews, you're going to need to use a little bit of imagination whilst looking at the storybook preview build side by side, as they aren't published yet